gateway-home/modules/ROOT/pages/index.adoc

```diff
@@ -1,53 +1,85 @@
-= Anypoint Platform Gateways
+= Anypoint Platform Gateways Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
+:page-aliases: gateway-home::gateway-overview.adoc
 
 An API gateway is an intermediary between a client application and a set of microservices or backend APIs. Responsibilities include request routing, rate limiting, caching, authentication, authorization, security, monitoring, and logging.
 
-A gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation concerns.
+A gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services to help you separate orchestration from implementation.
```

Comment:
enables

Reply:
(Good tightening here, by the way. I like that you keep just a little bit of generic, then launch into the two gateways.)

Reply:
Fixed